3rdparty/arc4random-src/arc4random_uniform.cpp

```diff
@@ -0,0 +1,84 @@
+/*	$OpenBSD: arc4random_uniform.c,v 1.2 2015/09/13 08:31:47 guenther Exp $	*/
+
+/*
+ * Copyright (c) 2008, Damien Miller <djm@openbsd.org>
```

Comment:
I don't think my changes really warrant claiming copyright to this.
There aren't any functional changes -- all changes are to make the file build.